test/mocha/helper/recurringEvents.test.js

```diff
@@ -46,9 +52,34 @@ describe('Recurring Event Helper tests', () => {
 		expect(recurringEventsHelper.getNumberForWeekday(i18nHelper.i18next.t('global.text.monday'), res)).to.equal(0);
 	});
 
-	it('createRecurringEvents', () => {
+	it('findAllWeekEvents', () => {
+		const startTime = '2017-10-20T10:00:00.000Z';
+		const endTime = '2017-10-20T10:45:00.000Z';
+		const until = '2017-11-21T00:00:00.000Z';
+		const weekDay = 'TH';
+
+		const recurringEvents = recurringEventsHelper.findAllWeekEvents(startTime, endTime, weekDay, until);
+		expect(timesHelper.dateToDateTimeString(recurringEvents[0].start)).to.be.equal('26.10.2017 12:00');
+		expect(timesHelper.dateToDateTimeString(recurringEvents[0].end)).to.be.equal('26.10.2017 12:45');
+
+		expect(timesHelper.dateToDateTimeString(recurringEvents[1].start)).to.equal('02.11.2017 12:00');
+		expect(timesHelper.dateToDateTimeString(recurringEvents[1].end)).to.equal('02.11.2017 12:45');
+	});
+
+	it('createRecurringEvents summer time change', () => {
```

Comment:
will this test still confirm what it needs to confirm when we are currently in summertime?

Reply:
in before section the default timezone is set to Berlin. In this test the times are so taken that the first events are in the summer time and  last events in the winter time. Winter time change in 2017 was on 29.10.2017

Reply:
I can add a comment to explain it.